.circleci/config.yml

```diff
@@ -4,26 +4,44 @@ jobs:
     machine: true
     steps:
       - checkout
+      - run:
+          name: Check for reserved branch names
+          command: |
+            if [[ $CIRCLE_BRANCH = latest || $CIRCLE_BRANCH = packer ]]; then
+              echo 'Can not use branches named "latest" or "packer" as these as meaningful tags.'
```

Comment:
This sentence doesn't make sense--I've edited to what I think you meant and fixed a typo:
`'Cannot use branches named "latest" or "packer" as these are meaningful tags.'`